### ISSUE TITLE
PHP warning: Warning: filemtime(): stat failed blockswp-share-block.php on line 73

### ISSUE THREAD

**Issue by hereliesvince** [Shelob9/blockswp-share-block#13]
The below error shows up for me site wide; in admin and frontend.

`Warning: filemtime(): stat failed for /srv/bindings/e9da6ac6fea94ffca57d26bb11a7d143/code/wp-content/plugins/blockswp-share-block/front.js in /srv/bindings/e9da6ac6fea94ffca57d26bb11a7d143/code/wp-content/plugins/blockswp-share-block/blockswp-share-block.php on line 73`

Conversation with Josh/Possible fix:

> Josh [8:23 PM] 
> Oh, that’s a bug in the plugin.
> josh [8:24 PM] 
> We should remove filemtime()

**Comment by easilyamused** [comment]
Tried fixing this and it lead to more problems. We thing something else is going on and this is a side effect or separate issues that's blocking a bigger issue. 

I was able to get the errors to go away but they it randomly brakes the editor in some environments. 

Rain this in nginx and apachy locally with flywheel and on patheon. On local it just dies. We tired wp debug but nothing was showing on crash.
